util/uvmdvgen/testplan.hjson.tpl

```diff
@@ -30,4 +30,11 @@
       tests: []
     }
   ]
+
+  covergroups: [
+    {
+      name: ${name}_feature_cg
+      desc: '''Describe the functionality covered by this covergroup.'''
```

Comment:
Does a non-existent item just cause a trivial report, or a reporting failure?

Reply:
Non-existent items will show up as "not implemented" - the onus is on the DV person to set the names correctly. 
Note that we still need to add that capability (reading tool generated reports and making a list of what's been implemented so far).